runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/DoFnOperator.java

```diff
@@ -285,7 +260,7 @@ public void open() throws Exception {
         mainOutputTag,
         additionalOutputTags,
         stepContext,
-        aggregatorFactory,
+        null,
```

Comment:
What happens if you pass `null` here. Shouldn't there simply be no parameter if aggregators are completely removed from Beam?